examples/transformerconfigs/README.md

```diff
@@ -1,37 +1,44 @@
 # Transformer Configurations
 
-Kustomize computes the resources by applying a series of transformers:
+Kustomize computes the configuration of a resource by applying a series of transformers:
+
 - namespace transformer
 - prefix/suffix transformer
 - label transformer
 - annotation transformer
 - name reference transformer
 - variable reference transformer
 
-Each transformer takes a list of resources and modifies certain fields. The modification is based on the transformer's rule.
-The fields to update is the transformer's configuration, which is a list of filedspec that can be represented in YAML format.
+Each transformer takes a list of resources and modifies certain fields in the resource based upon the transformer's configuration. A transformer's configuration is a list of `fieldSpec`, represented in YAML format.
 
-## fieldSpec
-FieldSpec is a type to represent a path to a field in one kind of resources. It has following format
-```
+## FieldSpec
+
+FieldSpec is a type that represents a path to a field in one kind of resource, such as `Job`.
```

Comment:
No need to mention Job here - it just confuses me.

Reply:
OK